Wikipedia/Code/WMFSearchLanguagesBarViewController.swift

```diff
@@ -121,6 +126,21 @@ class WMFSearchLanguagesBarViewController: UIViewController, WMFPreferredLanguag
         }
     }
     
+    fileprivate func showMoreLanguagesTooltipIfNecessary() {
+        guard !hidden && languageBarLanguages().count >= 2 && !UserDefaults.standard.wmf_didShowMoreLanguagesTooltip() else {
+            return
+        }
+        UserDefaults.standard.wmf_setDidShowMoreLanguagesTooltip(true)
+        self.perform(#selector(showMoreLanguagesTooltip), with: nil, afterDelay: 1.0)
```

Comment:
needs an equivalent `NSObject.cancelPreviousPerform...` call in `deinit` or `willDisappear` - otherwise it can be sent after this vc is deallocated, causing a crash. You could move the `wmf_setDidShow...` to `showMoreLangugaesTooltip` to ensure it's shown even if the call is cancelled.